library/src/main/java/com/airbnb/android/airmapview/WebViewMapFragment.java

```diff
@@ -254,6 +281,9 @@ public void run() {
             if (onMapLoadedListener != null) {
               onMapLoadedListener.onMapLoaded();
             }
+            if (onMapInitializedListener != null) {
```

Comment:
we should call `onMapInitialized` before `onMapLoaded` as that's how the native map will behave